vips/vip-xxx/bsctestnet.ts

```diff
@@ -0,0 +1,127 @@
+- MarketCapsRiskSteward can call _setMarketSupplyCaps on the BNB_CORE_COMPTROLLER
+- MarketCapsRiskSteward can call _setMarketBorrowCaps on the BNB_CORE_COMPTROLLER
+- MarketCapsRiskSteward can call setMarketSupplyCaps on IL comptrollers
+- MarketCapsRiskSteward can call setMarketBorrowCaps on IL comptrollers
+
+Risk Paramater Configs:
+- SupplyCap: RiskSteward: ${MARKET_CAPS_RISK_STEWARD}, Debounce: ${TEN_MINUTES}, Active: true
+- BorrowCap: RiskSteward: ${MARKET_CAPS_RISK_STEWARD}, Debounce: ${TEN_MINUTES}, Active: true
+
+VIP simulation: [https://github.com/VenusProtocol/vips/pull/459](https://github.com/VenusProtocol/vips/pull/XXX)`,
+    forDescription: "Execute this proposal",
+    againstDescription: "Do not execute this proposal",
+    abstainDescription: "Indifferent to execution",
+  };
+
+  return makeProposal(
+    [
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [RISK_STEWARD_RECEIVER, "setRiskParameterConfig(string,address,uint256)", NORMAL_TIMELOCK],
+      },
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [RISK_STEWARD_RECEIVER, "toggleConfigActive(string)", NORMAL_TIMELOCK],
+      },
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [RISK_STEWARD_RECEIVER, "pause()", NORMAL_TIMELOCK],
+      },
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [RISK_STEWARD_RECEIVER, "unpause()", NORMAL_TIMELOCK],
+      },
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [MARKET_CAPS_RISK_STEWARD, "processUpdate(RiskParameterUpdate)", RISK_STEWARD_RECEIVER],
+      },
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [MARKET_CAPS_RISK_STEWARD, "setMaxIncreaseBps(uint256)", NORMAL_TIMELOCK],
+      },
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [BNB_CORE_COMPTROLLER, "_setMarketSupplyCaps(address[],uint256[])", MARKET_CAPS_RISK_STEWARD],
+      },
+      {
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [BNB_CORE_COMPTROLLER, "_setMarketBorrowCaps(address[],uint256[])", MARKET_CAPS_RISK_STEWARD],
+      },
+      ...IL_COMPTROLLERS.map((comptroller) => ({
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [comptroller, "setMarketSupplyCaps(address[],uint256[])", MARKET_CAPS_RISK_STEWARD],
+      })),
+      ...IL_COMPTROLLERS.map((comptroller) => ({
+        target: ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [comptroller, "setMarketBorrowCaps(address[],uint256[])", MARKET_CAPS_RISK_STEWARD],
+      })),
```

Comment:
I think it would make sense to use the wildcard 0x0 in this case, allowing the risk steward contracts to invoke the mentioned functions on any contract. This way, we won't have to update the permissions if a new pool is added in the future